zombie-bite-scripts/report_account_migration_status.ts

```diff
@@ -112,17 +115,25 @@ async function fetchAndCacheAccounts(
     startKey = entries[entries.length - 1][0].toHex();
     const totalMinutes = Math.floor(totalElapsed / 60000);
     const totalSeconds = ((totalElapsed % 60000) / 1000).toFixed(1);
-    console.log(
-      `Fetched ${entries.length} accounts on page ${page}\nTotal accounts: ${totalFetched}\nTotal time: ${totalMinutes} minutes and ${totalSeconds} seconds`,
+    logger.info(
+      `Fetched ${entries.length} accounts on page ${page}`,
+      {
+        totalAccounts: totalFetched,
+        totalTime: `${totalMinutes} minutes and ${totalSeconds} seconds`
+      }
     );
 
     page++;
   }
-  console.timeEnd("Total fetching time");
-  console.log(`Finished fetching. Total accounts fetched: ${totalFetched}`);
+  
+  const totalTime = performance.now() - startTime;
+  logger.info("Finished fetching accounts", { 
```

Comment:
I am not quite sure, but worry these will be printed as `[Object Object]`. Have you tried one?